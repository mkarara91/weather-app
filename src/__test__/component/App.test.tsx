import React from 'react';
import {shallow} from "enzyme";
import {Provider} from 'react-redux';
import {store} from '../../app/store';
import App from '../../component/App';

test("app should match snapshot", () => {
    const c = shallow(<Provider store={store}>
        <App/>
    </Provider>);
    expect(c).toMatchSnapshot();
});
