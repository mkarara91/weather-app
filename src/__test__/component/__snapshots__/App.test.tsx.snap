// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app should match snapshot 1`] = `
<ContextProvider
  value={
    Object {
      "getServerState": undefined,
      "store": Object {
        "@@observable": [Function],
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "subscription": Object {
        "addNestedSub": [Function],
        "getListeners": [Function],
        "handleChangeWrapper": [Function],
        "isSubscribed": [Function],
        "notifyNestedSubs": [Function],
        "trySubscribe": [Function],
        "tryUnsubscribe": [Function],
      },
    }
  }
>
  <App />
</ContextProvider>
`;
